lib/diagnostic.rb

```diff
@@ -37,43 +37,46 @@
 # of multiplying by two the result of adding `a` and `b`. Name the method
 # conventionally.
 
-def method_name
+def twice_product(a, b)
+  (a + b) * 2
 end
 
-Diagnostic.defined_method = # call your method here
+Diagnostic.defined_method = twice_product(num1, num2)
 
 # Define a predicate method in Ruby
 #
 # Define a method that checks whether a number passed in is odd. Name it
 # conventionally.
 
-# define your method here
+def odd?(num)
+  num % 2 == 1
+end
 
-Diagnostic.predicate = # call your method here
+Diagnostic.predicate = 3.odd?
 
```

Comment:
so .odd? is already a method. be careful with this.  I see what you're trying to do though.